src/app/book/register/page.tsx

```
@@ -28,10 +30,78 @@ export default function BookRegister() {
   const statusSelect = useSingleSelect<ReadingStatus>('unread');
   const genresSelect = useMultiSelect<string>();
 
+  const [selectedFile, setSelectedFile] = useState<File | null>(null);
+  const [previewUrl, setPreviewUrl] = useState<string | null>(null);
+  const fileInputRef = useRef<HTMLInputElement>(null);
+
+  function handleOnImageChange(event: React.ChangeEvent<HTMLInputElement>) {
+    const file = event.target.files && event.target.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)